README.md

```diff
@@ -140,7 +140,7 @@ for dependency installation instructions for each supported operating system.
 1. Install third-party libraries:
 
     ```
-    sudo apt-get -y install cmake build-essential curl cppcheck g++-8 libbenchmark-dev libgflags-dev doxygen ruby-ronn libtinyxml2-dev libtinyxml-dev software-properties-common libeigen3-dev qtdeclarative5-models-plugin
+    sudo apt-get -y install cmake build-essential curl cppcheck g++-8 libbenchmark-dev libgflags-dev doxygen ruby-ronn libtinyxml2-dev libtinyxml-dev software-properties-common libeigen3-dev
```

Comment:
Now that every repository has all the dependencies listed in `.github/ci/packages.apt`, we should start using that in the installation instructions so the instructions are always up-to-date. What do you think of doing that here?

For example, running this after cloning:

```
export SYSTEM_VERSION=bionic
sudo apt -y install \
  $(sort -u $(find . -iname 'packages-'$SYSTEM_VERSION'.apt' -o -iname 'packages.apt') | tr '\n' ' ')
```

Reply:
This is definitely a cleaner approach that is easier to maintain. I went ahead and made this change in [a18718a](https://github.com/ignitionrobotics/ign-gazebo/pull/395/commits/a18718a30790768ccf0338798a02731657b5a804).

Reply:
Thanks. Just note that the user can only run that after cloning the repository, so that instruction should be moved down, or the cloning moved up.

Reply:
Just noticed that as well. I went ahead and made a note about cloning first in [4ae6846](https://github.com/ignitionrobotics/ign-gazebo/pull/395/commits/4ae684628272c11a392ca89254b42d7930987341).